docs/index.rst

```diff
@@ -6,16 +6,6 @@
 Welcome to AMD MIGraphX's documentation!
 ========================================
 
-.. toctree::
-   :maxdepth: 3
-   :caption: Contents:
-
-   py_user_guide
-   cpp_user_guide
-   driver
-   contributor_guide
```

Comment:
I assume this is removed because of _toc.yml.in? Should this file be removed as well?

Reply:
Yes, this was to prevent having them duplicated in the table of contents. I kept this since a short description of the project and the indices seemed useful for a homepage.